docs/guides/src/main/operator/installation.adoc

```diff
@@ -5,14 +5,48 @@
 
 <@tmpl.guide
 title="Keycloak Operator Installation"
-summary="How to install the Keycloak Operator on Kubernetes">
-
-TODO: https://github.com/keycloak/keycloak/issues/10786
+summary="How to install the Keycloak Operator on Kubernetes and OpenShift">
 
 == Keycloak Operator Installation
+In this guide we will show how to install the Keycloak Operator in your Kubernetes or OpenShift cluster.
+
+=== OLM Installation
+
+You can install the Keycloak Operator in Kubernetes environment is to use Operator Lifecycle Manager (OLM).
+Before moving on, make sure you have OLM installed, if not follow this https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#install-a-release[guide].
+
+To install the Keycloak Operator follow the https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/[instructions].
```

Comment:
```suggestion
The Keycloak Operator OLM package can be installed from the OLM catalog. For general instructions on how to install operators using OLM, follow the https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/[instructions] on the OLM page.
```

Reply:
Not sure if we should not add a rough doc on our side though, as we'Re pointing to OLM often. Then again we would have to maintain changes on their side, so works for me.